### DISCUSSION TITLE
Support blob url for downloading files

### DISCUSSION THREAD

**Discussion by StatusSceptre** [status-im/status-legacy#16639] [General]
# Bug Report

## Problem 
App  shows an error and crashes on when attempting to download specific file.
![Screenshot_20220705-073513_Status](https://user-images.githubusercontent.com/42685581/177273062-0b62178c-c021-4836-b501-3f9004b26d1d.jpg)

Other pdf files download just fine. Details for this pdf
![Screenshot 2022-07-05 093826](https://user-images.githubusercontent.com/42685581/177276119-608fb345-7d28-4c91-a5d1-61729d944607.png)

#### Expected behavior
PDF opens in browser and/or file is downloaded

#### Actual behavior
App throws error and crashes
Handle pdf not working for this type of pdf?

### Reproduction

Unsure how to reproduce with specific pdf.

### Additional Information

- Status version: 1.19
- Operating System: Android
- Webview version 103.0.5060.70

[comment]: # (Please, add logs/notes if necessary)

**Comment by hesterbruikman** [comment]
(by @hesterbruikman )

**Comment by ibrkhalil** [comment]
Truns out this happens with all the links that starts with something other than `http` or `https`
Like for instance, `blob:// `
`react-native-webview` doesn't support these kinds of urls yet
